scripts/run.sh

```diff
@@ -55,6 +55,9 @@ echo ENABLE_SOLIDITY_TESTS: ${ENABLE_SOLIDITY_TESTS}
 echo GINKGO_SKIP_FLAGS: ${GINKGO_SKIP_FLAGS}
 echo AVALANCHE_LOG_LEVEL: ${AVALANCHE_LOG_LEVEL}
 
+# avoid blst related invalid operation errors
+export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
```

Comment:
could we move these flags into `constants.sh` and instead import it in `run.sh` and `build_test.sh`?

Reply:
already did. maybe you are looking at a previous version of the code?

Reply:
yup my bad, thanks!